### DISCUSSION TITLE
Sign up for the Keyfactor Community Newsletter

### DISCUSSION THREAD

**Discussion by KarolinHem** [Keyfactor/signserver-ce#4] [Announcements]
Do you want to stay up to date on the latest news, software updates, hands-on tutorials and user workshops on all Keyfactor’s open-source software?  

Welcome to visit [Keyfactor Community](https://www.keyfactor.com/community/) and sign up for our **Newsletter**.

**Comment by ramu96** [comment]
bandi.ramumca08@gmail.com
